tmp/build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -0,0 +1,207 @@
+    mediatype: image/png
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+      - serviceAccountName: istio-operator
+        rules:
+__CLUSTER_ROLE_RULES__
+      deployments:
+      - name: istio-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: istio-operator
+          template:
+            metadata:
+              labels:
+                name: istio-operator
+            spec:
+              serviceAccountName: istio-operator
+              volumes:
+              - name: discovery-cache
+                emptyDir:
+                  medium: Memory
+              containers:
+                - name: istio-operator
+                  image: __IMAGE_SRC__
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  command:
+                  - istio-operator
+                  - --discoveryCacheDir
+                  - /home/istio-operator/.kube/cache/discovery
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      value: ""
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: "istio-operator"
+                  volumeMounts:
+                  - name: discovery-cache
+                    mountPath: /home/istio-operator/.kube/cache/discovery
+  customresourcedefinitions:
+    required:
```

Comment:
Is this ok to add now or should we wait until we figure out how to manage dependencies?

Reply:
It's a small enough change if we decide to remove it. My understanding is our operator relies on kiali and jager CRDs being present too.